Gems/Atom/RPI/Code/Include/Atom/RPI.Edit/Common/JsonUtils.h

```diff
@@ -20,6 +20,10 @@ namespace AZ
     {
         namespace JsonUtils
         {
+            //! Protects from allocating too much memory. The choice of a 1MB threshold is arbitrary.
+            //! If you need to work with larger files, please use AZ::IO directly instead of these utility functions.
+            inline constexpr size_t AtomMaxFileSize = 1024 * 1024;
```

Comment:
Nitpick: We rarely (if ever) use the word "Atom" in the code. Personally, I would call this AZ::RPI::DefaultMaxJsonFileSize.

Reply:
Saw this after the PR is merged. Created a new PR to fix this : https://github.com/o3de/o3de/pull/4067
Used 'DefaultMaxFileSize' instead of 'DefaultMaxJsonFileSize' since it's used by both ReadFile() and ReadJsonFile()